recipes-browser/cog/files/cog-drm-kill-weston

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
-sudo systemctl stop weston.service
```

Comment:
Did you try this?  I cannot test it quickly right now.  I don't think that it works by default as a regular user in most systems (e.g. desktop) for system-wide units.  But I'm happy to try it and change it to "su -c" or similar if it doesn't work.

Reply:
The idea is execute it as `root` if you need to do it.

Reply:
ACK, I was thinking that it might be called from some of the icons in the Weston panel but this is not the case.